autoload/sneak/label.vim

```diff
@@ -135,6 +143,8 @@ func! s:before() abort
   endif
 
   syntax clear
```

Comment:
If we're still using `syntax clear` I don't see the purpose, it doesn't address https://github.com/justinmk/vim-sneak/issues/136

Reply:
Right, hadn't read the thread toroughly.